### ISSUE TITLE
Midterm Peer Review–cq39

### ISSUE THREAD

**Issue by christopherqian164** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#7]
The project is about analyzing several different algorithms for updating the weights in neural networks, specifically looking at their accuracy and time. They use the MNIST and CIFAR-10 datasets, as well as a stock return dataset. The goal is to develop a method to identify an optimal learning rate. 

Things I liked:
1. It defines each of the methods that it tests.
2. The results are presented in tables and graphs for easy interpretation.
3. Care was taken for when feature engineering for the stock data to get the best results. 

Things that could use improvement:
1. The paper talks a lot about the importance for picking a good learning rate, but I don't see any results for actually picking one (in fact, no learning rate is listed for what produced the results)
2. I'm not sure what is being predicted with the stock data. Is it the price? Or correlation?
3. I'd like to see some discussion on why one particular method works better than the others.
